packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/Chart.yaml.patch

```diff
@@ -8,7 +8,7 @@
 +  catalog.cattle.io/certified: rancher
 +  catalog.cattle.io/deploys-on-os: windows
 +  catalog.cattle.io/display-name: Monitoring
-+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.28.0-0'
++  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.29.0-0'
```

Comment:
I think we want this annotation to specify >= 1.23 and < 1.29. From the tagged issues:

> Update kube-version annotation in Chart.yaml to catalog.cattle.io/kube-version: '>= 1.23.0-0' or catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.29.0-0'